girder/models/model_base.py

```diff
@@ -1330,7 +1330,7 @@ def textSearch(self, query, user=None, filters=None, limit=0, offset=0,
             cursor, user=user, level=level, limit=limit, offset=offset)
 
     def prefixSearch(self, query, user=None, filters=None, limit=0, offset=0,
-                     sort=None, fields=None, level=AccessType.READ):
+                     sort=None, fields=None, prefixSearchFields=None, level=AccessType.READ):
```

Comment:
Let's put this new kwarg last since otherwise it would theoretically break backward compatibility.